### ISSUE TITLE
Add legends to Langmuir plot examples

### ISSUE THREAD

**Issue by lemmatum** [PlasmaPy/PlasmaPy#456]
The plots [here](http://docs.plasmapy.org/en/stable/auto_examples/plot_langmuir_analysis.html#sphx-glr-auto-examples-plot-langmuir-analysis-py) could use some nice legends.

**Comment by manasbedmutha98** [comment]
I tried this in a local cloned version by executing examples/plot_langmuir_analysis.py, post making edits in the plotting. Even with/without those edits I am not able to see the plots. However it does print the required details when swept_probe_analysis() is called.
Also, I tried looking up for auto_examples under the docs, but it is set to be ignored via the .gitignore file. I checked my env and installation but couldn't any problem there. Can anybody help out why I can't see any plots?

**Comment by ritiek** [comment]
> Further, despite having astropy v3.0.2 initially, I didn't get any error like #467.

By any chance you're using astropy v3.0.3? It seems like it got fixed it the newer version but I was able to still reproduce it with astropy v3.0.2.

> I tried this in a local cloned version by executing examples/plot_langmuir_analysis.py, post making edits in the plotting. Even with/without those edits I am not able to see the plots.

I don't know what's the agreed way to show them when executing them from the terminal but adding these lines at the end (after the code to plot the graph) seems to work for me.
```python
>>> import matplotlib.pyplot as plt
>>> plt.show()
```

If you're using jupyter notebook, you should be able to see them by adding
```python
%matplotlib inline
```
at the top of your code.

**Comment by manasbedmutha98** [comment]
I checked it again. It was v3.0.2 only. Previously, the function call never went to actually plot it, hence o error. I did get the said error after that. 
Thanks for pointing out the plt.show(), I'll create a PR for this issue soon.

**Comment by ritiek** [comment]
> I checked it again. It was v3.0.2 only.

That's weird but I think it is actually a good thing that it doesn't error out. :D

> Thanks for pointing out the plt.show(), I'll create a PR for this issue soon.

You're welcome. That said, we'll still need to confirm what's the preferred way to show plots. Maybe @StanczakDominik or @jasperbeckers who wrote the code?
